website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -242,7 +242,10 @@ For incremental models, like SQL models, you will need to filter incoming tables
 import snowflake.snowpark.functions as F
 
 def model(dbt, session):
-    dbt.config(materialized = "incremental")
+    dbt.config(
+        materialized = "incremental",
+        unique_key = "id",
```

Comment:
should the comma after "id" be there?

Reply:
The trailing comma is valid 👍

I put it in there because it is consistent with the type of [formatting that Black would do](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas), and it makes the `diff`s more simple if/when a new config item is added.